test_suite/unicode_bmp1.json

```diff
@@ -0,0 +1 @@
+["\uD83D\uDCD8"]
```

Comment:
1) It should be `unicode_bmp1.jsonnet`, not `.json`. It's parsed and treated as Jsonnet, so `.json` extension is a bit misleading here.
2) We'll need more examples like that which exercise our BMP-detection.